src/App.jsx

```diff
@@ -7,19 +7,35 @@ import database from './database.json';
 import './App.css';
 
 function App() {
+  const [productsList, setProductsList] = useState(database);
```

Comment:
Keep a copy of the `database.json` file in-memory so we are able to perform "write" and "read" operations on it.